examples/basic-wallet/src/lib.rs

```diff
@@ -4,7 +4,7 @@
 // However, we could still use some standard library types while
 // remaining no-std compatible, if we uncommented the following lines:
 //
-extern crate alloc;
```

Comment:
Why is this commented out? Seems to me that if you don't have the alloc crate linked, then we don't need to define a `#[global_allocator]` either. But also, we may as well remove the commented-out line as well if we don't actually need to link liballoc.

Reply:
I see examples as a starting point for our users to build their own code. This is why I want to leave it along with the comment. I commented out `extern crate alloc;` since it's not needed for this example code.

`#[global_allocator]` is required for no_std. Otherwise the following error is thrown:
```
error: no global memory allocator found but one is required; link to std or add `#[global_allocator]` to a static item that implements the GlobalAlloc trait
```